site/docs/master/configuration.md

```diff
@@ -1,9 +1,54 @@
 # Contour Configuration
 
+## Configuration File
```

Comment:
nit: Maybe just change the title to be `Contour Configuration File`

Reply:
Since this page is all for Contour, I felt that it was clear. I'm planning to add similar sections for command line flags and environment variables. Do you think that titling all of these "Contour Foo" would get too repetitive?

Reply:
Next push, I'm going to rename this page to "Contour Configuration Reference", WDYT?